pkg/daemon/daemon.go

```diff
@@ -544,6 +544,14 @@ func (dn *Daemon) RunFirstbootCompleteMachineconfig() error {
 		return errors.Wrap(err, "failed to rename encapsulated MachineConfig after processing on firstboot")
 	}
 
+	tuningChanged, err := UpdateTuningArgs(KernelTuningFile, CmdLineFile)
```

Comment:
A bit late to see this PR. Having this functionality here is going to work only during firstboot. Isn't it that users who maybe using this functionality would want to add/delete these kargs later on as well? Since, these are not applied using MachineConfig, there is no standard way to undo changes done during firstboot.

Reply:
Right, removing default kargs via MCs should be solved in https://issues.redhat.com/browse/GRPA-2490

Reply:
sorry for the noise, I missed that we already do it at https://github.com/openshift/machine-config-operator/pull/1942/files#diff-06b532a2a5a5d3ad1d5341b3e00945efR547